postgrest_py/_async/client.py

```diff
@@ -20,16 +20,20 @@ def __init__(
         *,
         schema: str = "public",
         headers: Dict[str, str] = DEFAULT_POSTGREST_CLIENT_HEADERS,
+        timeout: Timeout = Timeout(5),
```

Comment:
/nit should we set a constant for the timeout duration and use that here?

Reply:
+1